### ISSUE TITLE
is there any google maps version working with microg?

### ISSUE THREAD

**Issue by patrickdrd** [microg/android_packages_apps_GmsCore#709]
so has anyone managed to make google maps work with microg?
if so, which version and how?
I tried with many of them a couple of months ago,
but none worked for me (I'm running microg services official build),
so I had to switch to here maps, but I hate it because I have to download 0,5gb
(my whole country, while I'm only interested on my city and such) each time and
seems a bit more sluggish compared to google maps too I guess

**Comment by mar-v-in** [comment]
In my experience, older versions work fine. I tested 9.75.1

**Comment by patrickdrd** [comment]
ok, I installed 9.82.1 and it worked for now (it recognized my account too and I downloaded offline maps as well), but I have to test it on the next days too, because the previous time it wasn't working with mobile data just when I needed it too, I'm keeping here maps too for the time being

**Comment by Riamse** [comment]
Latest Google Maps works flawlessly if you do not log into your Google account.

**Comment by patrickdrd** [comment]
yes, it does, I updated and checked, but even with the latest, there's a problem:
you can't use pedestrian mode with offline maps, I mean what about when there's no signal reception for example?
not a microg fault of course, but since I'm mostly pedestrian these days, I'm keeping here maps,
which works with offline maps (no signal etc)

**Comment by andyboeh** [comment]
Pedestrian mode reports no signal, but works nevertheless for me, i.e. it correctly reports turns etc.

**Comment by frknkrc44** [comment]
Update: latest version (10.80.0) works fine but I installed it on 9.82.1
